### DISCUSSION TITLE
Route not found when polling | component is within another namespace

### DISCUSSION THREAD

**Discussion by dyanakiev** [livewire/livewire#1228] [4. General]
Hello, im using nWidart/laravel-modules and my modules are in app/Modules/Admin,Pages,News etc...
In Modules views im using `@livewire('App\Modules\Admin\Http\Livewire\ClockInline')` and its displaying the component just fine, but polling is not working, thats the only thing i tested now, maybe other action doesnt work too. As you see the component is stored there: `App\Modules\Admin\Http\Livewire\ClockInline`.

In config the class_namespace is `App\\Http\\Livewire` with is default and i dont think i need to change, polling livewires from the default namespace works fine.

Am i doing anything wrong or that is just not supported yet?

```
Livewire\Exceptions\ComponentNotFoundException
Unable to find component: [app.modules.admin.http.livewire.clock-inline]
https://example.com/livewire/message/app.modules.admin.http.livewire.clock-inline

```

Note: manually registering this from the module service provider works fine, i guess they are just not discoverable when not manually registered.
```
Livewire::component('ClockInline', ClockInline::class);
```

**Comment by dkonsoftware** [comment]
Hi same here.

Seems like **Livewire::component('component-name', Component::class);** works only in tests for "non Route" components. Otherwise component name is taken from discovery file.
